controllers/news.controllers.js

```diff
@@ -41,15 +45,35 @@ exports.patchArticleById = (req, res, next) => {
     .catch(next)
 }
 
-exports.getUsers = (req, res) => {
-  selectUsers().then((users) => {
-    res.status(200).send({ users })
-  })
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((users) => {
+      res.status(200).send({ users })
+    })
+    .catch(next)
 }
 
-exports.getArticles = (req, res) => {
+/* exports.getArticles = (req, res, next) => {
   selectArticles().then((articles) => {
-    console.log(articles, '<<<<<<<<<')
-    res.status(200).send({ articles })
+    const promises = []
+    articles.forEach((article) => {
+      promises.push(commentsByArticle(article.article_id))
+    })
+    Promise.all(promises)
+      .then((results) => {
+        let articleWithComments = articles.map((article, index) => {
+          article.comment_count = Number(results[index].count)
```

Comment:
should be doing get articles with comment count in one query here, not using a promise.all. Shouldn't use the length, should take advantage of SQL functions.